website/static/js/osfHelpers.js

```diff
@@ -838,6 +838,17 @@ var findContribName = function (userAttributes) {
     }
 };
 
+// Check if a user is in the list of project contributors returned by the v2 API.
+function userIsContributor(userId, contributors) {
```

Comment:
@sloria, can I use `lodash.find` instead of implementing a similar `_.find` method?